person.rb

```diff
@@ -1,17 +1,16 @@
 require './nameable'
 
-# class for person data
 class Person < Nameable
-  # class for person data
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Creating a teacher or a student is currently proving buggy. Please take a look at the screenshots below:
  
  Creating a teacher
  
  ![image](https://user-images.githubusercontent.com/45002654/169666678-24636e39-04e1-4653-9232-d1d2c7cec3c3.png)
  
  Creating  a student
  
  ![image](https://user-images.githubusercontent.com/45002654/169666717-2150b5ed-36b5-4a43-9c22-d5d566444d21.png)
  
  The culprit is how we are setting the value of `parent_permission`. Instead of the `:` we need to use an assignment operator `=` to set the value of `parent_permission`
  
  ```suggestion
    def initialize(age, name = 'Unknown', parent_permission = true)
  ```
  We'll need to do the same refactoring in our Student and Teacher classes.
  
  Please work on this change so that one can be able to create a student and a teacher,

Reply:
@Lameck1 I´m trying to apply your suggestion, but linter is NOT allowing me to... please see the result below:
![image](https://user-images.githubusercontent.com/77974883/169667381-7663b716-11da-408b-8da5-dab327725a55.png)

Please, any suggestion on this

Reply:
Thanks for the review... I could identify that the error was related to the number of arguments and the way I was previously passing them.